### DISCUSSION TITLE
Nästa Milestone

### DISCUSSION THREAD

**Discussion by Tenkaklet** [Tenkaklet/coderdojo#14] [Ideas]
### Vad blir nästa steg?
"lets start a conversation" :)

**Comment by Tenkaklet** [comment]
Jag tänker att vad jag kan göra är att set up en domän och digital ocean?
-_först_ tycker jag vi ska ha en grundkod i repot dock.

**Comment by Oskar-Sjoberg-dev** [comment]
Jag tänker för att få en konkret riktning för projektet så skulle man kunna höra med Dimitrij om vad han/dom han pratar med efterfrågar.
För att underlätta för Dimitrij samt göra det snabbt och smidigt så kan vi ha förberett koncept som vi tror är genomförbara med den tid/erfarenhet vi tillsammans har.

Tex:
* efterfrågas det hjälp i form av information? -> skapa en sida där man kan enkelt navigera information.
* efterfrågas det hjälp i form av säker/pricksäker överföring av donationer -> skapa en sida som matchar kriterierna.
* efterfrågas det nått annat/har vi andra idéer? -> möta det?

Efter det steget tänker jag att det blir lättare att bestämma och komma igång med design. 

Hur känner ni om det som tillväga gång?

// tänkte främst från design perspektiv av sidan (är sämre på hur backend strukturerar).

**Comment by Tenkaklet** [comment]
Detta är super bra input och jag kommer fråga dessa frågor till Dimitrij och ha det dokumenterat 🥇
